release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -819,6 +828,61 @@ module openconfig-if-ip {
     }
   }
 
+  grouping ip-vrrp-simple-key-authentication-config {
+    description
+      "This grouping defines VRRP simple authentication config.";
+
+    leaf auth-mode {
```

Comment:
@dplore Doesn't this leaf do the same job as `auth-type` that you'd mentioned? (maybe it was added after you commented.)